pipeline/terraform/main.tf

```diff
@@ -1,8 +1,33 @@
-module "catalogue_pipeline_20201111" {
+locals {
+  stacks = {
+    "20201111" = {
+      release_label     = "prod"
+      enable_reindexing = false
+    }
+
+    # A new pipeline with transformers that include support for multiple languages.
+    # See https://github.com/wellcomecollection/platform/issues/4864
+    "2020-11-12" = {
```

Comment:
Minor: I find the eight-digit dates quite difficult to read, and spent a not insignificant amount of time not noticing the difference between `20201111` and `20200111`. This is an attempt to make these a little more readable.

Reply:
I'm not 100% certain that we don't have any code that does a `index_name.split('-')` or equivalent - can't think of or find any examples but something to be aware of

Reply:
I ran this yesterday and it populated the index fine, so 🤞 it's an issue.

(That was before I realised I'd not put in the key change. 🤦‍♀️ )